index.html

```diff
@@ -27,11 +27,19 @@
           <a href="index.html"
             ><img src="media/Union.svg" alt="menu icon" class="hamburger"
           /></a>
+          <ul id="nav-links">
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+          </ul>
         </nav>
       </header>
       <div class="intro">
```

Comment:
- [ ] In **Desktop version**, The `top section` should have the same design as the original website. Kindly fix the bottom-left design. Hint You can do that by using `border-radius` for design.
- Your website 
![ssdg](https://user-images.githubusercontent.com/26767383/154678426-5d9e9a14-009b-44b1-ba0f-069fb0a98051.PNG)
- Original website 
![ffdvfv](https://user-images.githubusercontent.com/26767383/154678456-7feb0b94-38be-4261-b722-1d62a32c760d.PNG)